controller/users_credentials_controller.py

```diff
@@ -0,0 +1,27 @@
+from model.repository.users_credentials_repository import UsersCredentialsRepository
```

Comment:
This file is no longer needed since we'll go directly from AppController to the Repositories